src/experiments.js

```diff
@@ -341,6 +341,9 @@ class ABTestBlock extends Test {
 				},
 			}, false );
 		} );
+
+		// Dispatch the altisblockcontentchanged event.
+		window.dispatchEvent( new Event( 'altisBlockContentChanged' ) );
```

Comment:
This is coming after quite a few return statements in the function, the content is changed further when `innerHTML` is set to empty and the template is appended as a child node. This means currently the block content could change but this event would not be dispatched.